packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-timeline-item.tsx

```diff
@@ -93,7 +95,7 @@ const ItemHeader: FC<ItemHeaderProps> = memo(
           </DropdownMenu.Trigger>
           <DropdownMenu.Content className="w-[216px]" align="end">
             <DropdownMenu.Group>
-              {!hideEditDelete ? <DropdownMenu.Item onClick={onEditClick}>Edit</DropdownMenu.Item> : null}
+              {!!hideEditDelete && <DropdownMenu.Item onClick={onEditClick}>Edit</DropdownMenu.Item>}
```

Comment:
Does this flip the logic to show when `hideEditDelete` is truthy?
```suggestion
              {!hideEditDelete && <DropdownMenu.Item onClick={onEditClick}>Edit</DropdownMenu.Item>}
```